packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -180,19 +180,21 @@ const SchedulePreviewBody = decorate([
             />
           </Tooltip>
         ) : (
-          <ActionButton
-            btnStyle='primary'
-            data-id={job.id}
-            data-name={job.name}
-            data-nVms={nVms}
-            data-schedule={schedule.id}
-            data-type={job.type}
-            handler={_runBackupJob}
-            icon='run-schedule'
-            key='run'
-            size='small'
-          />
-        )}{' '}
+          <Tooltip content={_('runBackupJob')}>
```

Comment:
You can pass the tooltip to the `ActionButton`. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/action-button.js#L56